### PR TITLE
Translate multiple vulnerabilities in RubyGems (zh_tw)

### PR DESCRIPTION
**Pull request by bugtender**
zh_tw Translation for multiple vulnerabilities in RubyGems

- zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md

Thank you.

### CODE REVIEW

File: zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: news_post
+title: "RubyGems 發現多個漏洞"
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: How about: `RubyGems 多個安全性風險`?  I have seen more usage than `漏洞`。

File: zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## 詳情
+
+以下為已回報的漏洞
+
+* DNS 請求劫持漏洞 (CVE-2017-0902)
+* ANSI 轉義序列漏洞 (CVE-2017-0899)
+* `query` 命令中的 DoS 漏洞 (CVE-2017-0900)
+* 存在於 gem 安裝程式中，一個能讓惡意 gem 隨意複寫檔案的漏洞 (CVE-2017-0901)
+
+強烈建議 Ruby 使用者儘速採用採用下列的因應措施。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: `採用採用` => `採用`

Comment (bugtender) [reply zh_tw/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: Fixed ! Thanks for reviewing.

**Comment by JuanitoFatas** [comment]
Thanks for your contribution ❤️

**Pull request merged**